tests/unit_tests/cli/snapshots/test_integration_cli/test_es_mda/es_mda_integration_snapshot

```diff
@@ -1,11 +1,21 @@
-,Realization,COEFFS:COEFF_A,COEFFS:COEFF_B,COEFFS:COEFF_C
+Iteration,Realization,COEFFS:COEFF_A,COEFFS:COEFF_B,COEFFS:COEFF_C
 iter-0,1,0.366973404074,1.13199917643,1.17249316965
 iter-0,2,0.896799325213,1.26410484631,0.0223414803805
 iter-0,4,0.95469117332,0.754759801257,2.08805559758
 iter-0,8,0.0982839315406,1.26664483158,2.49804909664
 iter-0,16,0.306190596539,1.30051410642,0.795822702625
-iter-1,1,0.479832969234,1.05074551321,1.57574613433
-iter-1,2,0.646826348358,1.12000512395,0.991469828792
-iter-1,4,0.747341831225,0.810829509581,3.46933364221
-iter-1,8,0.360231725305,1.15996928007,1.94295928927
-iter-1,16,0.514843048849,0.966771660378,3.33748200281
+iter-1,1,0.348999794806,1.17883866345,0.876723482266
```

Comment:
Why are the numbers different?

Reply:
I extended it from 1 iteration with `weigths=1` to three iterations with the default weight, meaning that the old update was run with a different global_std_scaling